docs/source/en/quicktour.mdx

```diff
@@ -425,6 +387,53 @@ Create a model from your custom configuration with [`TFAutoModel.from_config`]:
 
 Take a look at the [Create a custom architecture](./create_a_model) guide for more information about building custom configurations.
 
+## PyTorch Trainer
+
+All models are a standard [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) or a [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) so you can use them in any standard training loop. While you can write your own training loop, 🤗 Transformers provides a [`Trainer`] class for PyTorch, which contains the basic training loop and adds additional functionality for features like distributed training, mixed precision, and more.
+
+<Tip>
+
+For TensorFlow, use [`~TFPreTrainedModel.prepare_tf_dataset`] to prepare your dataset as a `tf.data.Dataset`. Then you can [`compile`](https://keras.io/api/models/model_training_apis/#compile-method) and [`fit`](https://keras.io/api/models/model_training_apis/#fit-method) your model with the usual Keras methods. Take a look at the [finetuning tutorial](./training#convert-dataset-to-tensorflow-format) for more details.
+
+</Tip>
```

Comment:
I don't know if I'd read this in this section; I'm in a PyTorch section, so if I'm a TensorFlow user, I feel like I wouldn't look at this :smile: